src/app/cases/styles.tsx

```diff
@@ -40,3 +49,18 @@ export const Body = styled.div`
   grid-template-columns: 4.5fr 10fr;
   overflow: hidden;
 `;
+
+export const ResetFilters = styled.button`
```

Comment:
could we possibly make this bigger? the text feels a bit tiny

Reply:
bro i realized... it only looks tiny on safari 🤠 

safari:
<img width="830" alt="Screenshot 2023-11-23 at 7 07 25 PM" src="https://github.com/calblueprint/immigration-justice-project/assets/47801900/982e1bf8-6d87-425e-913c-cb6b393de18b">

chrome:
<img width="847" alt="Screenshot 2023-11-23 at 7 07 34 PM" src="https://github.com/calblueprint/immigration-justice-project/assets/47801900/ab2c7ac8-6e58-48b9-8faa-650d78c83457">

Reply:
safari is trying too hard to be different

Reply:
i added a `font-size` prop to the styling, which should (hopefully) fix it

Reply:
looks good now!! tysm